### ISSUE TITLE
2016 links

### ISSUE THREAD

**Issue by sinaneza** [STAT545-UBC/Discussion#498]
Hi 

For people who had the course stat545 in 2016, links for that time are more indicative. I would appreciate if we could still have access to those

Best
Sina Nezafatkhah

@jennybc 
@oganm

**Comment by jennybc** [comment]
@sinaneza Our plan is to keep substantive content at stable links, but that landing pages for class meetings and homework assignments, etc. are perishable. They can change from year to year. Can you be more specific about what links are changing that you think should be stable, in light of that info?

**Comment by sinaneza** [comment]
Hi Professor

Sorry for my delayed answer. 

I would appreciate if I could have access to you tutorial on dplyr functions, dplyr join functions, ggplot, tidying data, stringr and regular expression, shiny app, and automation

**Comment by jennybc** [comment]
As far as I can tell, that is all still in the usual location, no?

**Comment by sinaneza** [comment]
Hi Professor

The syllabus section has massively changed, but now I noticed that they are available in Topic section. However I am not sure all of them are available. I will see them in detail and if I couldn't find a material will contact you soon

**Comment by jennybc** [comment]
Yes, the syllabus section is intended to change with each run. It has time-sensitive content, such as homework. The Topics page is more stable. I've discussed this with the 2017 instructors, so I think we're all on the same page about what is stable and what is not.
